app/models/like.rb

```diff
@@ -1,2 +1,4 @@
 class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :comment
```

Comment:
- [ ] following the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) the likes belong only to the user and post ,kindly fix that :pray: 

example : 


```ruby
belongs_to :user
belongs_to :post
```